src/lang/wasm.ts

```diff
@@ -581,11 +581,11 @@ export const executor = async (
   try {
     let jsAppSettings = default_app_settings()
     if (!TEST) {
-      const lastSettingsSnapshot = await import(
-        'components/SettingsAuthProvider'
-      ).then((module) => module.lastSettingsContextSnapshot)
-      if (lastSettingsSnapshot) {
-        jsAppSettings = getAllCurrentSettings(lastSettingsSnapshot)
+      const settings = await import('machines/appMachine').then((module) =>
+        module.getSettings()
+      )
```

Comment:
I'm noticing some sort of logical cyclic ... relationship going on here and back in index.tsx? How come we need to import the module _twice_ across the app?